### PR TITLE
Adding srtp1

### PR DESCRIPTION
**Pull request by leedm777**
This formula was copied from homebrew-core at
e3d6d868156ccc9d7592db3206d165712710d8da:Formula/srtp.rb

### CODE REVIEW

**Comment by apjanke** [comment]
@BrewTestBot test this please

File: srtp1.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  sha256 "86e1efe353397c0751f6bdd709794143bd1b76494412860f16ff2b6d9c304eda"
+
+  bottle do
+    cellar :any
+    revision 1
+    sha256 "d5f214f13d8c34c39241cd68cb201f72a4fbb493e6a8694d965b2f86b30b8015" => :el_capitan
+    sha256 "7a9ac7216782ef2ff6e351705a627eaf535c412fb668086868ab86e5a8334413" => :yosemite
+    sha256 "28561181099811365178f954b2938c7921dfcd5cdbf2e5129dbf4ae3fb15b910" => :mavericks
+    sha256 "bc26d7076d2cb0c219ab498e2760153ed22b6703be69a5120c114be9d8e5ee2a" => :mountain_lion
+  end
```
Comment (MikeMcQuaid) [inline srtp1.rb]: Remove the bottles, they will be added when we merge.

File: srtp1.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class Srtp1 < Formula
```
Comment (MikeMcQuaid) [inline srtp1.rb]: Go for `Srtp15`

**Comment by MikeMcQuaid** [comment]
We need a test added. Ideally, can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by leedm777** [comment]
Test added; bottles removed; renamed to srtp15.

**Comment by leedm777** [comment]
@MikeMcQuaid Look good now?

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**